### PR TITLE
changes rev deconversion text color

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
title, it changes it from red to purple

## Why It's Good For The Game
being deconverted from revolutions no longer blends in with the red attack logs
Capture

## Changelog
:cl: SuicidalPickles
tweak: Changed the revolutionary deconversion message to purple font, to keep it from blending with attack logs.
/:cl:

### CODE REVIEW

**Comment by bigfatbananacyclops** [comment]
maybe add an effect on screen, sometimes people never look on the right side of text and keep attacking even after deconverted

**Comment by Angustmeta** [comment]
The text itself can be confusing to new players, as you don't really see a clear message specifically saying "Person X has brainwashed you" when revved. Then they can get called retards and told stuff like "you are now on my shitlist" when they get deconverted and can't figure out who converted them.

**Review by ExcessiveUseOfCobblestone — DISMISSED** [review]
make it a class

File: code/modules/antagonists/revolution/revolution.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 /datum/antagonist/rev/farewell()
 	if(ishuman(owner.current) || ismonkey(owner.current))
 		owner.current.visible_message("<span class='deconversion_message'>[owner.current] looks like [owner.current.p_theyve()] just remembered [owner.current.p_their()] real allegiance!</span>", null, null, null, owner.current)
-		to_chat(owner, "<span class='userdanger'>You are no longer a brainwashed revolutionary! Your memory is hazy from the time you were a rebel...the only thing you remember is the name of the one who brainwashed you...</span>")
+		to_chat(owner, "<span class ='userdanger'><B><font color=purple>You are no longer a brainwashed revolutionary! Your memory is hazy from the time you were a rebel...the only thing you remember is the name of the one who brainwashed you....</font></B> </span>")
```
Comment (ShizCalev) [inline code/modules/antagonists/revolution/revolution.dm]:
```suggestion
		to_chat(owner, "<span class ='deconversion_message'>You are no longer a brainwashed revolutionary! Your memory is hazy from the time you were a rebel...the only thing you remember is the name of the one who brainwashed you....</span>")
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**